src/test/java/lotto/domain/LottoGameTest.java

```diff
@@ -0,0 +1,76 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import lotto.view.OutputView;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LottoGameTest {
+
+    LottoGame lottoGame;
+
+    @BeforeEach
+    void 초기화(){
+        lottoGame = new LottoGame(
+            Arrays.asList(
+                new LottoLine("1, 2, 3, 4, 5, 6"),
+                new LottoLine("4, 12, 23, 34, 41, 45"),
+                new LottoLine("1, 15, 18, 24, 35, 39"),
+                new LottoLine("11, 17, 28, 32, 43, 45"),
+                new LottoLine("3, 7, 23, 24, 29, 36"),
+                new LottoLine("1, 22, 23, 35, 39, 45")
+            )
+        );
+    }
+
+    @Test
+    @DisplayName("주어진 개수만큼 로또 생성 테스트")
+    void 로또게임_생성_테스트(){
+        LottoGame lottoGameByCount = LottoGame.issueLotto(14);
+        Assertions.assertThat(lottoGameByCount.toLottoGameDTO().size()).isEqualTo(14);
+    }
+
+    @Test
+    @DisplayName("생성된 로또 문자열 테스트")
+    void 로또게임_출력_테스트(){
+        OutputView outputView = new OutputView();
+        Assertions.assertThat(outputView.getLottoGameString(lottoGame.toLottoGameDTO()))
+            .isEqualTo(
+                "[1, 2, 3, 4, 5, 6]\n"
+                + "[4, 12, 23, 34, 41, 45]\n"
+                + "[1, 15, 18, 24, 35, 39]\n"
+                + "[11, 17, 28, 32, 43, 45]\n"
+                + "[3, 7, 23, 24, 29, 36]\n"
+                + "[1, 22, 23, 35, 39, 45]"
+            );
+    }
+
+    @Test
+    @DisplayName("생성된 로또 문자열 출력 테스트")
+    void 로또게임_결과_테스트(){
+        OutputView outputView = new OutputView();
+        LottoResult lottoResult = lottoGame.getLottoResult(new LottoLine("1, 17, 23, 35, 39, 45"));
+        Assertions.assertThat(outputView.getLottoResultString(lottoResult))
+            .isEqualTo(
+                "3개 일치 (5000원)- 1개\n"
+                + "4개 일치 (50000원)- 0개\n"
+                + "5개 일치 (1500000원)- 1개\n"
+                + "6개 일치 (2000000000원)- 0개\n"
+        );
+    }
+
+    @Test
+    @DisplayName("생성된 로또 수익률 문자열 출력 테스트")
+    void 로또게임_수익률_테스트(){
+        OutputView outputView = new OutputView();
+        LottoResult lottoResult = lottoGame.getLottoResult(new LottoLine("1, 17, 23, 35, 39, 45"));
+        LottoPayment payment = new LottoPayment(14000);
+        LottoPayment prize = new LottoPayment(lottoResult.getLottoPrize());
+        Assertions.assertThat(outputView.getEarningRateString(payment.toLottoPaymentDTO(), prize.toLottoPaymentDTO()))
+            .isEqualTo(
+                "총 수익률은 107.5입니다."
+            );
+    }
```

Comment:
테스트코드도 가독성이 매우 중요하다고 생각하는데요 이미 아시는 내용이겠지만 `given` `when` `then` 을 구분해주시면 좋을 것 같습니다

추가로 저는 `when`, `then`을 먼저 작성후에 `when`에 필요한 `given`을 제일 마지막에 작성해주는 편이에요!

```suggestion
        // given
        OutputView outputView = new OutputView();
        LottoResult lottoResult = lottoGame.getLottoResult(new LottoLine("1, 17, 23, 35, 39, 45"));
        LottoPayment payment = new LottoPayment(14000);
        LottoPayment prize = new LottoPayment(lottoResult.getLottoPrize());
        
        // when
        String expectedMessage = outputView.getEarningRateString(payment.toLottoPaymentDTO(), prize.toLottoPaymentDTO());
        
        // then
        Assertions.assertThat(expectedMessage).isEqualTo("총 수익률은 107.5입니다.");
    }
```

Reply:
넵 테스트 코드 작성시에 given, when, then은 형식적인 주석이라고 생각해서 달지 않고 있었는데 명시하도록 해보겠습니다